### ISSUE TITLE
Coordinates not being reprojected

### ISSUE THREAD

**Issue by jflasher** [developmentseed/sentinel-api-data#2]
It looks like for some records, the coordinates are not being reprojected. Coordinates are being returned like

```
{"type":"Feature","geometry":{"crs":{"type":"name","properties":{"name":"urn:ogc:def:crs:EPSG:8.8.1:32623"}},"type":"Polygon","coordinates":[[[499980,7100040],[609780,7100040],[609780,6990240],[499980,6990240],[499980,7100040]]]},"properties":{}}
```

**Comment by scisco** [comment]
@jflasher I haven't been able to find such records, but will look out for it. Do you have the logs by any chance?

**Comment by jflasher** [comment]
Nope, no logs, sorry. Will keep an eye out if I see it again and try to capture something.

**Comment by drewbo** [comment]
To confirm @jflasher, this causes the errors in this other open source project that we both contribute to because then the mapbox upload fails? https://github.com/AstroDigital/ad-sentinel-vector-tile-updater

Could we also temporarily solve it by ignoring these records?

**Comment by jflasher** [comment]
@drewbo yep, I think that's correct, we get an out of bounds error for the lat/lon on upload. Could probably just do a check around https://github.com/AstroDigital/ad-sentinel-vector-tile-updater/blob/master/updater/updater.js#L124 and ignore if it's not in proper lat/lon. I may be able to do this at some point next week, but not in the next couple of days.
